tests/kernel/device/src/main.c

```diff
@@ -33,7 +33,7 @@
  * information and validates its binding.
  *
  * @see device_get_binding(), device_busy_set(), device_busy_clear(),
- * DEVICE_AND_API_INIT()
+ * DEVICE_AND_API_INIT(NULL, )
```

Comment:
I guess the changes in this file are unintentional.

Reply:
Yes. Thank you.

Reply:
Fixed